src/release_archive.bzl

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def release_archive(name, srcs = [], src_map = {}, package_dir = "-", deps = [], visibility = []):
```

Comment:
Nit: Maybe `source_archive` is a better name?

Reply:
It contains jars with class files, so it's not only for source files. I'd prefer to keep the name as they use it in rules_kotlin, so that it is easier to find.

Reply:
OK, sounds good.